spec/pluto/format/jpeg_spec.cr

```diff
@@ -1,6 +1,6 @@
 require "../../spec_helper"
 
-describe Pluto::Format::JPEG do
+Spectator.describe Pluto::Format::JPEG do
```

Comment:
Instead of a global scope `describe` macro, spectator prefers the top level `describe` to use the module macro directly. All subsequent `describe` and `context` can continue being used as they are.